apps/kampus/app/pano/features/post/PanoFilterLink.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+export function PanoFilterLink({ href, children }: { href: string; children: React.ReactNode }) {
```

Comment:
let's convert this to a util & component that accepts the filter type (can be a string literal union), and returns an absolute url based on that.